modules/auxiliary/scanner/http/trace.rb

```diff
@@ -27,36 +31,36 @@ def run_host(target_host)
     begin
       res = send_request_raw({
         'version'      => '1.0',
-        'uri'          => '/',
+        'uri'          => '/<script>alert(1337)</script>', #XST Payload
         'method'       => 'TRACE',
         'headers' =>
         {
-          'Cookie' => "did you echo me back?",
+          'Cookie' => "TRACE ME PLS",
         },
-      }, 10)
+      })
 
       if res.nil?
-        print_error("no repsonse for #{target_host}")
```

Comment:
You will have to return at this point. Also, rather than if then return, use unless. 
```ruby
unless res
  fail_with(Failure::Unreacheable, 'No response received from the target host')
```